source/tools/entry.js

```diff
@@ -6,6 +6,16 @@ const EntryProperty = {
     Username:                     "username"
 };
 
+const EntryDefaultAttribute = {
+    PassModifiedTime:              "passModifiedTime",
+    PassExpiryTime:                "passExpiryTime",
+    PassExpiry:                    "passExpiry"
```

Comment:
I think these keys need only be exposed in the Entry.Attributes object. Also, I would still say that it's better not seeing any defaults. `0` or _not present_ simply mean the same thing. Like I said: all currently users don't have any data here currently, so they will be of the _not present_ type. I don't think having a default value adds much value to the feature.